trie/utils/verkle.go

```diff
@@ -144,7 +144,7 @@ func GetTreeKeyVersion(address []byte) []byte {
 }
```

Comment:
I'll be making some changes today to optimize some `uint256.Bytes()` calls in this file.
Not related with these PRs since they exist from before; but just giving a hint.